<?xml version="1.0"?>

<launch>
  
  <node pkg="hector_stability_assistance" type="stability_visualization_node" name="stability_visualization" output="screen" respawn="true">
    <param name="update_frequency" value="10"/>
    <param name="predict_pose" value="false"/>
    <param name="nominal_stability_margin" value="2.0"/>

    <!-- sdf contact estimation lib parameters -->
    <param name="pose_predictor" value="sdf_contact_estimation::SDFContactEstimation"/>
    <rosparam file="$(find hector_stability_assistance)/config/sdf_contact_estimation.yaml" command="load" ns="sdf_pose_predictor"/>
    <rosparam file="$(find sdf_contact_estimation)/config/asterix_shape_config.yaml" command="load" ns="sdf_pose_predictor/shape_model" />
    <param name="sdf_pose_predictor/debug" value="false"/>

    <!-- esdf server parameters -->
    <rosparam file="$(find hector_sensor_proc_launch)/config/voxblox_common_config.yaml" command="load" ns="esdf_server"/>
    <param name="esdf_server/update_mesh_every_n_sec" value="0.0"/>
    <remap from="~/esdf_server/esdf_map_in" to="/esdf_server/esdf_map_out" />
  </node>

 
</launch>
